packages/likert/package.json

```diff
@@ -9,14 +9,17 @@
   "module": "src/index.js",
   "dependencies": {
     "@material-ui/core": "^3.9.2",
-    "@pie-framework/pie-player-events": "^0.1.0",
-    "@pie-lib/graphing": "^2.0.0",
+    "@pie-lib/config-ui": "^10.8.25",
```

Comment:
you shouldn't depend on config-ui in a ui package - can you remove this?

Reply:
solved